app/views/drawings/index.html.erb

```diff
@@ -30,36 +31,41 @@
   </div>
 </div>
 
-<div class="current-drawings">
-  <% filter_current(@drawings).each_slice(2) do |drawings| %>
-    <div class="govuk-grid-row">
-      <% drawings.each.with_index do |drawing, index| %>
-        <div class="govuk-grid-column-one-half">
-          <div class="drawing-card<%= " right" if index % 2 == 1 %>">
-            <p class="govuk-body">
-              <strong><%= drawing.name %></strong><br/>
-              <%= drawing.created_at.strftime("%e %B %Y") %><br/>
-              <%= link_to "View in new window",
-              rails_blob_path(drawing.plan), target: :_new %><br/>
-            </p>
+<ul class="app-task-list current-drawings">
```

Comment:
Unless I'm missing something, we've removed current-drawings from the custom CSS, so should we remove it from the <ul>?

Reply:
It's being used by the tests, which isn't a great reason, but leaving semantic(ish) classes in the HTML doesn't upset me too much